src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+var React = require('React');
+var ReactDOMFiber = require('ReactDOMFiber');
+
+describe('ReactDOMFiber', () => {
+  var container;
+
+  beforeEach(() => {
+    // to supress a warning that ReactDOMFiber is an experimental renderer.
+    spyOn(console, 'error');
```

Comment:
I like the idea of just importing `ReactDOM` and wrapping it all in a `ReactDOMFeatureFlags.useFiber`.